esmf_regrid/experimental/io.py

```diff
@@ -69,36 +74,57 @@ def save_regridder(rg, filename):
         )
         raise TypeError(msg)
 
-    metadata_name = "regridder weights and metadata"
-
     weight_matrix = rg.regridder.weight_matrix
     reformatted_weight_matrix = weight_matrix.tocoo()
     weight_data = reformatted_weight_matrix.data
     weight_rows = reformatted_weight_matrix.row
     weight_cols = reformatted_weight_matrix.col
     weight_shape = reformatted_weight_matrix.shape
 
+    esmf_version = rg.regridder.esmf_version
+    esmf_regrid_version = rg.regridder.esmf_regrid_version
+    save_version = esmf_regrid.__version__
+
+    # Currently, all schemes use the fracarea normalization.
+    normalization = "fracarea"
+
     mdtol = rg.mdtol
     attributes = {
-        "regridder type": regridder_type,
+        "title": "iris-esmf-regrid regridding scheme",
+        "regridder_type": regridder_type,
+        "ESMF_version": esmf_version,
+        "esmf_regrid_version_on_initialise": esmf_regrid_version,
+        "esmf_regrid_version_on_save": save_version,
+        "normalization": normalization,
```

Comment:
`"regridder_type"`, `"ESMF_version"`, `"esmf_regrid_version_on_initialise"` and `"mdtol"` are all referenced in multiple places so need the module constant treatment too.